src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -533,17 +534,22 @@ public function dropColumn($tableName, $columnName)
      * Get an array of indexes from a particular table.
      *
      * @param string $tableName Table Name
+     * @param boolean $lowercase
      * @return array
      */
-    protected function getIndexes($tableName)
+    public function getIndexes($tableName, $lowercase = true)
```

Comment:
this method was originally lowercasing all index column names (as it is currently).  I initially thought I needed to have case sensitive column names for mysql on some operating systems.  That turned out to be incorrect thinking on my part.  I have removed the extra param